src/bin/tikv-importer.rs

```diff
@@ -93,6 +94,14 @@ fn main() {
 
     let config = setup_config(&matches);
 
+    // FIXME: initial_logger() should white-list tikv_importer!
+    if let Some(mut targets) = env::var_os("TIKV_EXTRA_LOG_TARGETS") {
+        targets.push(",tikv_importer");
+        env::set_var("TIKV_EXTRA_LOG_TARGETS", targets);
```

Comment:
TiKV has change to a black list and it output all logs by default.

https://github.com/tikv/tikv/pull/5174/files#diff-a0c2af9b9008a1c91d7459a7ff1302bbR42